stemcell_builder/stages/prepare_rawdisk_image_stemcell/apply.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2009-2012 VMware, Inc.
+
+set -e
+
+base_dir=$(readlink -nf $(dirname $0)/../..)
+source $base_dir/lib/prelude_apply.bash
+
+pushd $work
+rm -f disk.raw
+ln ${stemcell_image_name} disk.raw
+tar zcf stemcell/image disk.raw
+popd
```

Comment:
seems like this is doing the exact same thing as https://github.com/cloudfoundry/bosh/blob/master/stemcell_builder/stages/prepare_raw_image_stemcell/apply.sh aside from name; perhaps existing raw type could be used?